packages/patternfly-4/react-topology/package.json

```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "@patternfly/patternfly": "2.65.0",
     "@patternfly/react-core": "^3.140.6",
```

Comment:
this one also?

Reply:
actually i do see a prerelease with 3.140.6 in npm

Reply:
Is this al set?  This is not a version i changed, it appears to be the latest pre-release though

Reply:
Yes I think this looks correct, the latest prerelease is 3.140.6 - the other places where this changes (react-inline-edit-extension/package.json and react-table/package.json) should be reverted back to match this 3.140.6 version number